domain/storage/service/storagepool.go

```diff
@@ -306,24 +315,38 @@ func (s *StoragePoolService) ListStoragePoolsByProviders(
 	return pools, nil
 }
 
-// GetStoragePoolByName returns the storage pool with the specified name.
+// GetStoragePoolUUID returns the UUID of the storage pool with the specified name.
 // The following errors can be expected:
 // - [storageerrors.PoolNotFoundError] if a pool with the specified name does not exist.
-func (s *StoragePoolService) GetStoragePoolByName(ctx context.Context, name string) (domainstorage.StoragePool, error) {
+func (s *StoragePoolService) GetStoragePoolUUID(ctx context.Context, name string) (domainstorage.StoragePoolUUID, error) {
```

Comment:
We should add the `ByName` component to the func name to be consistent with other implementation?